test/e2e/run_cleanup_test.go

```diff
@@ -35,7 +35,7 @@ var _ = Describe("Podman run exit", func() {
 
 	It("podman run -d mount cleanup test", func() {
 		SkipIfRemote("podman-remote does not support mount")
-		SkipIfRootless("FIXME podman mount requires podman unshare first")
+		SkipIfRootless("TODO rootless podman mount requires podman unshare first")
```

Comment:
Changed FIXME to TODO. The difference, as I see it, is that TODO is "that would take effort. Maybe some day we'll enhance the test". Whereas FIXME is "this is broken". (And my `find-obsolete-skips` script flags FIXME but not TODO)